network.go

```diff
@@ -97,15 +117,15 @@ func (network *_Network) _SetNetworkName(net NetworkName) {
 	network._SetLedgerID(*ledger)
 }
 
-func _ReadAddressBookResource(ad string) map[AccountID]NodeAddress {
+func _ReadAddressBookResource(ad string) (map[AccountID]NodeAddress, error) {
 	f, err := ioutil.ReadFile(ad)
 	if err != nil {
-		panic(err)
+		return nil, err
```

Comment:
the change avoids panic but doesn't really fix the bug. the function only works when there happens to be a `addressbook/xxxnet.pb` file on disk.

I'm not sure how the tests can pass before. most likely, the test application / case is run in the same directory of the sdk source, that's why it can read the file.

Reply:
Fixed this the same we fixed it in the JS SDK which is to include the hex for each address book within the SDK. This isn't a great fix, but its the only fix I could come up with because Go doesn't have a "resources/" directory like Java does for published packages :/